<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Имитатор ИКД</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="time_mode_box">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Регулярные импульсы</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Случайные импульсы</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="time_model_box">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Постоянная</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Линейная</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Экспоненциальная</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Экспоненциальная с изменяющимся периодом</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Модель реактора</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="model_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Скорость счета</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="model_param_table">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>371</width>
      <height>241</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Значение</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="start_gen_file_but">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сгенерировать файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_generation">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_generation">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Стоп</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="freq_line">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>401</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <row>
     <property name="text">
      <string>Форма импульса</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Длительность импульса, нс</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Новая строка</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Средний заряд, Кл</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Пиковый ток в среднем импульсе, А</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Пиковое напряжение в среднем импульсе, В</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Математическое ожидание сигнала тока, А</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Математическое ожидание сигнала напряжение, В</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>СКО сигнала тока, А</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>СКО сигнала напряжения, В</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Значение</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Конфигурация</string>
    </property>
    <addaction name="form_conf"/>
    <addaction name="spec_conf"/>
    <addaction name="options"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="options">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="form_conf">
   <property name="text">
    <string>Форма импульса</string>
   </property>
  </action>
  <action name="spec_conf">
   <property name="text">
    <string>Спектр зарядов</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
